### PR TITLE
Fix warnings reported by LGTM.com

### PR DESCRIPTION
**Pull request by Scrumplex**

### CODE REVIEW

**Comment by Scrumplex** [comment]
> This pull request **introduces 1 alert** and **fixes 7** when merging [9ec2606](https://github.com/PolyMC/PolyMC/commit/9ec260619b48447e398445aecd6651d319b8217e) into [03e454b](https://github.com/PolyMC/PolyMC/commit/03e454b71d83aebbb534c5734ccd9093842da28c) - [view on LGTM.com](https://lgtm.com/projects/g/PolyMC/PolyMC/rev/pr-e058de696af5630520df98963e6bd16962183692)
> 
> **new alerts:**
> 
>     * 1 for FIXME comment
> 
> 
> **fixed alerts:**
> 
>     * 4 for Throwing pointers
> 
>     * 1 for Resource not released in destructor
> 
>     * 1 for Constant return type on member
> 
>     * 1 for Comparison result is always the same

You win some, you lose some

**Comment by RaptaG** [comment]
> You win some, you lose some

… it’s all a game to me

File: launcher/translations/POTranslator.h [hunk 1]
```diff
@@ -9,6 +9,7 @@ class POTranslator : public QTranslator
     Q_OBJECT
 public:
     explicit POTranslator(const QString& filename, QObject * parent = nullptr);
+    virtual ~POTranslator();
```
Comment (LennyMcLennington) [inline launcher/translations/POTranslator.h:12]: Couldn't d just be changed to a smart pointer instead of manually defining a destructor

Comment (Scrumplex) [reply launcher/translations/POTranslator.h:12]: There is no reason to make this a smart pointer as its existence is tied to a `POTranslator` instance.

**Review by flowln — APPROVED** [review]

**Review by ZekeZDev — APPROVED** [review]

**Review by txtsd — APPROVED** [review]
LGTM!

**Pull request merged**